### PR TITLE
Button Component with Test

### LINKED ISSUE #88: Test button group component
## Context
Every component must also have adequate unit testing and regression testing

## Problem
Issue #86 does not include testing

## Solution
- [ ] Complete a suite of unit tests for the button group component (issue #86)

### PR DESCRIPTION
**Pull request by DemiAdisa**
## Source 

Closes #88 
Closes Prior Button Component Issue

### Summary of PR 

This PR includes a Button Component and a test to go with it. 

### Purpose 

To include the Button Component for re-use in our application.

### Review and Testing Steps

- [ ] Pull this branch
- [ ] Read the accompanying documentation. It can be found in the app\client\mindjogg\components\StdButton\ directory
- [ ] Run "npm install"
- [ ] Run "npm start", you should see a mockup button on the first screen, it should say "Sign Up"
- [ ] Click it and see if it navigates to a different page.
- [ ] Head over to app\client\mindjogg\screens\acoount_management\SignInScreen.js and play around with the StdButtonComponent and see if it still renders after changing the props.
- [ ] Leave review
- [ ] Run "npm test" and ensure all the tests pass.

## Additional Information (If applicable)
Nil

### Notes
In the test for the button, I tried to test that the button executes the callback when clicked but I was encountering issues so it will be something to go back to and implement. You can take a look at it in the app\client\mindjogg\components\StdButton\StdButton.test.js file and if you can see where I was going wrong, please help out. Thanks.

### Resources 
Used the official Jest documentation [here](https://jestjs.io/docs/tutorial-react-native)

### CODE REVIEW

**Review by jyaneshseeam — APPROVED** [review]
Test is passed

File: app/client/mindjogg/components/StdButton/StdButton.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          buttonPress={onClickCallback}
+        />
+      )
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+});
+
+// describe("Standard Text Button", () => {
```
Comment (ja-chisholm) [inline app/client/mindjogg/components/StdButton/StdButton.test.js:26]: I think you should copy/paste this test for longPress as well

**Review by ja-chisholm — APPROVED** [review]
All looks good

File: app/client/mindjogg/components/StdButton/StdButton.test.js (hunk 1 above)
Comment (DemiAdisa) [reply app/client/mindjogg/components/StdButton/StdButton.test.js:26]: Ah I can do that in a later PR 👍, will need to make an issue for it

Comment (ja-chisholm) [reply app/client/mindjogg/components/StdButton/StdButton.test.js:26]: wicked sounds good

**Pull request merged**